Sources/Toast.swift

```diff
@@ -111,7 +111,11 @@ open class Toast: Operation {
     DispatchQueue.main.async {
       self.view.setNeedsLayout()
       self.view.alpha = 0
-      ToastWindow.shared.addSubview(self.view)
+        
+      let window = UIApplication.shared.windows[UIApplication.shared.windows.count - 1]
```

Comment:
please replace with `UIApplication.shared.windows.last` to avoid race issues.

Reply:
Hi, I replaced the suggested code.